### DISCUSSION TITLE
Inspirarse en código obtenido en internet

### DISCUSSION THREAD

**Discussion by jalonsoluc** [IIC2133-PUC/2024-1#17] [Tarea 0] [answered]
Hola!

Puedo hacer una parte de mi código inspirado en otro código que encontré en internet? Por supuesto que citándolo correctamente, tanto comentado en el código como en el README.md.

Gracias!
Juan José

**Comment by CataAlamos** [comment, marked as answer]
Hola! Mientras cites correctamente tanto en el código como el Readme no hay problema.

**Reply by jalonsoluc** [reply]
Gracias Cata!
